src/docs/v10-vs-v11-retrospective.md

```diff
@@ -0,0 +1,643 @@
+# Merging the Berkeley schema into the NMDC schema
+
+This document is a retrospective on the changes made to the NMDC schema between major versions 10 and 11. The
+development of version 11 was launched during a meeting at the Lawrence Berkeley National Laboratory in December 2023,
+and the development took place in a fork of the nmdc-schema called berkeley-schema-fy24. This is the first time a major
+update had comprehensive planning and development in a fork. A subsequent update in the same style has not planned yet
+but is not ruled out.
+
+Motivations for the Berkeley schema development included:
+
+- First class representations of "omics" methods beyond nucleic acid analysis, like mass-spectrometry based
+  proteomics and metabolomics. Within the Berkeley schema development team, this effort has gone by names like
+  "replacing `OmicsProcessing` with `DataGeneration`".
+- Renaming and refactoring of the processes that model the processing of data, i.e. bioinformatics workflows.
+- A continuation of bipartite (material->process->material->process) modeling for the processing of samples, following
+  the bipartite modeling used in the bioinformatics workflows (data->process->data).
+- Continuation of using example data files as unit tests of the schema
+- Switching numerous string-range slots to enumeration-range slots.
+- A requirement that all data records assert their own type and the addition of support for polymorphic aggregations of
+  data
+- Establishing schema development and modeling best practices and sharing domain understanding between subteams
+
+See also: https://github.com/microbiomedata/nmdc-schema/releases/tag/v11.0.0
+
+## Conventions:
+
+- `this back-ticked font` is used to indicate the verbatim names of elements from the schema and variables, functions,
```

Comment:
```suggestion
- `this monospaced font` is used to indicate the verbatim names of elements from the schema and variables, functions,
```

I don't think the backticks will be visible in rendered views of the document. I recommend describing it differently (e.g. "monospaced", "stylized").

Reply:
The adjective "back-ticked" appears elsewhere in this document.